packages/shared-state-async/files/usr/share/rpcd/acl.d/shared-state-async.json

```diff
@@ -0,0 +1,15 @@
+{
+    "lime-app": {
+        "description": "lime-app public access",
```

Comment:
why do we explicitely mention lime-app here? other components could benefit to access shared-state-async via ubus too

Reply:
While logged in via ssh, you have direct, full access to ubus. This file is to enable lime app to use it. other users should enable other profiles.